antora-ui-camel/src/css/frontpage.css

```diff
@@ -1,212 +1,200 @@
-header.frontpage {
-  font-size: 1.5rem;
-  display: flex;
-  flex-flow: column;
-  align-items: center;
-  margin: 0 auto;
-  max-width: 60rem;
-}
+/* common css for sections */
 
-header.frontpage h1 {
-  font-size: 5rem;
-  font-weight: bolder;
+section.frontpage h1,
+section.frontpage h2,
+section.frontpage h3,
+section.frontpage h4,
+section.frontpage h5,
+section.frontpage h6 {
+  font-weight: var(--heading-font-weight);
   color: var(--heading-font-color);
-  margin-top: 3rem;
-  text-align: center;
-  text-transform: none;
-  text-shadow: -1px -1px 1px #fff5, 1px -1px 1px #fff5, -1px 1px 1px #fff5, 1px 1px 1px #fff5;
+  line-height: 1.3;
+  margin: 1rem 0 0;
+  text-transform: uppercase;
+  word-wrap: break-word;
 }
 
-header.frontpage p {
-  hyphens: none;
-  margin-top: 3rem;
-  text-align: center;
-  line-height: 1.5;
-  color: var(--body-font-color);
+section.frontpage {
+  color: var(--doc-font-color);
+  font-size: var(--doc-font-size);
+  line-height: var(--doc-line-height);
+  margin: 0 auto;
+  max-width: var(--frontpage-max-width);
 }
 
-header.frontpage a {
-  color: var(--heading-font-color);
-  text-decoration: underline;
```

Comment:
I think it's a mistake that we don't underline links by default. I created [a separate issue for that](https://issues.apache.org/jira/browse/CAMEL-15371).